deployment/ec2/amis.py

```diff
@@ -0,0 +1,40 @@
+"""Helper functions to handle EC2 related operations with Boto3"""
+
+import boto3
+import logging
+
+
+LOGGER = logging.getLogger(__file__)
+LOGGER.addHandler(logging.StreamHandler())
+LOGGER.setLevel(logging.INFO)
+
+
+def _prune_ami(ec2, ami):
+    """Actually deregister AMI and its associated snapshot"""
+    LOGGER.info('Identified that [%s] is eligible to be pruned..',
+                ami["ImageId"])
+
+    ec2.deregister_image(ImageId=ami["ImageId"])
```

Comment:
Fixed in [36dee9c](https://github.com/azavea/raster-vision/pull/93/commits/36dee9cf969457c718e5f9e8b4c72a26bd09c0ac)